src/platform/backends/qemu/check_kvm_support

```diff
@@ -64,9 +88,9 @@ if [ "$disabled" -eq 1 ]; then
     exit 1
 fi
 
-modprobe $kvm_mod
+modprobe $kvm_mod > /dev/null 2>&1
 
 check_kvm_module
 
-echo "Unable to load '$kvm_mod'."
+echo "Unable to load kvm support. Please ensure kvm is installed on your machine."
```

Comment:
> you capitalize KVM in other messages above. Why not here?

Oops, I will do that.

> Also, can KVM actually be installed?

Well, it's a kernel module, so perhaps it's not compiled in the kernel being used?  I'm not sure how else to word that without being too technical.

> Checking the return code of the modprobe, and if failure, printing its error message would help people debug the issue. And us, if people paste that output into a bug.

I originally had the output of `modprobe`, but @Saviq pointed out that it's not what we want in #811 and https://github.com/CanonicalLtd/multipass/pull/948#issuecomment-52119838. So I'm not sure how else to handle that other to be generic.

Reply:
Ok, pushed the change for capitalizing 'KVM'.

Reply:
You mean @Saviq 's comment "Could we wrap this in a better error message?" ?

Could we compromise by printing the modprobe error to stderr, and setting `QProcess::ForwardedErrorChannel` so that it'll end up in the journal?

I just hate throwing away a useful error message.